src/Service/Geocoder.php

```diff
@@ -52,7 +52,7 @@ public function encode(Address $address): array
         });
 
         if (is_null($item)) {
-            throw new GeocoderException('Unable to look up address');
+            throw new GeocoderException(sprintf('Unable to look up address: %d', $address->getId() ?? -1));
```

Comment:
A textual representation (rather than an integer) of the address will be more useful in this message.

Reply:
Fixed